### DISCUSSION TITLE
[Files] Can directories be toggled open/close?

### DISCUSSION THREAD

**Discussion by practicalli-johnny** [jesseduffield/lazygit#2632] [General]
In the Files view, it it possible to toggle a directory to its content is either shown or hidden?

I have repositories that are undergoing some major refactor work, which have a long list of changes contained in many different directories.  It would be easier to navigate the changes if directories could be collapsed when not being worked on.

Thank you.

**Comment by stefanhaller** [comment]
Just hit `enter`.
